packages/react-native-executorch/common/rnexecutorch/models/ocr/RecognitionHandler.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <opencv2/opencv.hpp>
+#include <rnexecutorch/models/ocr/CTCLabelConverter.h>
+#include <rnexecutorch/models/ocr/Recognizer.h>
+#include <rnexecutorch/models/ocr/Types.h>
+#include <string>
+#include <vector>
+
+namespace rnexecutorch {
+
+class RecognitionHandler {
+public:
+  RecognitionHandler(std::string recognizerSourceLarge,
+                     std::string recognizerSourceMedium,
+                     std::string recognizerSourceSmall, std::string symbols,
+                     std::shared_ptr<react::CallInvoker> callInvoker);
+  std::vector<OCRDetection> recognize(std::vector<DetectorBBox> bboxesList,
+                                      cv::Mat &imgGray, cv::Size desiredSize);
+  void unload();
+  std::size_t getMemoryLowerBound();
```

Comment:
And change in source
```suggestion
  std::size_t getMemoryLowerBound() const noexcept;
```